charts/auth-layer-server/README.md

```diff
@@ -0,0 +1,245 @@
+# HederaTheGraph (HTG) Authentication Layer Server
+
+The Authentication Layer Server is a KeyCloak server that provides authentication and authorization services for the HederaTheGraph (HTG). It is based on the [KeyCloak](https://www.keycloak.org/) open source identity and access management solution. 
+With the following configurations:
+- Realm preconfigured with the HTG (htg-auth-layer) client
+- Groups and Roles preconfigured for the HTG client
+- Custom claim mappers for the HTG client
+- Token expiration time set to 10 hours (plenty of time for a one day work session)
+- Disabled refresh tokens
+- Other custom configurations
+    - User registration is disabled
+    - Forgot password is enabled
+    - Email as username is enabled
+    - Login with email is enabled
+
+**For more information about KeyCloak Configuration, please visit the [KeyCloak Documentation](https://www.keycloak.org/docs/latest/server_admin/index.html)**
+
+## Prerequisites
+- Minikube or a Kubernetes cluster [(more here)](https://minikube.sigs.k8s.io/docs/start/)
+- Helm 3 [(install instructions here)](https://helm.sh/docs/intro/install/)
+- kubectl [(install instructions here)](https://kubernetes.io/docs/tasks/tools/)
+
+## Installation
+
+To install the Authentication Layer Server, run the following commands:
+
+- Install the chart, using the custom values file provided in the `values.yaml` file
+```bash
+helm install htg-auth-server .
+```
+
+### Helm Chart Overrides
+
+#### Admin Password
+
+Define a custom password, use `auth.adminPassword` like this: 
+```bash
+helm install htg-auth-server . --set auth.adminPassword=yourpassword
```

Comment:
This did not work for me - I installed via this command: `helm install htg-auth-server . --set auth.adminPassword=secret`.

I was able to successfully retrieve the admin password from the secret, so maybe it's just not setting the correct variable?

Reply:
Fixed, it was actually `keycloak.auth.adminPassword=yourpassword`

✅